commit c5f355b77a5f11f7a75e7de2e485fab25ad638df
Author: Christophe Mutricy <xtophe@videolan.org>
Date:   Sun Apr 5 22:19:52 2009 +0100

    Revert "Safe default ACL also for the old HTTP interface"
    
    This was the opposite of what was intended
    
    This reverts commit 1d82f475bda7f36a3d1927ab9a1d9afeb6a0df80.

Index: vlc-0.9.9a/share/http/.hosts
===================================================================
--- vlc-0.9.9a.orig/share/http/.hosts
+++ vlc-0.9.9a/share/http/.hosts
@@ -1,5 +1,5 @@
 #
-# Default access-list for VLC HTTP interface
+# Access-list for VLC HTTP interface
 # $Id: 320_all_http_acl.patch,v 1.1 2009/05/10 13:51:45 aballier Exp $
 #
 
@@ -16,9 +16,9 @@
 #10.0.0.0/8
 #172.16.0.0/12
 #192.168.0.0/16
+#169.254.0.0/16
 
-# The world
-# (comment this out to obtain a safe default)
-::/0
-0.0.0.0/0
+# The world (uncommenting these 2 lines is not quite safe)
+#::/0
+#0.0.0.0/0
 
Index: vlc-0.9.9a/share/http/old/.hosts
===================================================================
--- vlc-0.9.9a.orig/share/http/old/.hosts
+++ vlc-0.9.9a/share/http/old/.hosts
@@ -1,5 +1,5 @@
 #
-# Default access-list for VLC HTTP interface
+# Access-list for VLC HTTP interface
 # $Id: 320_all_http_acl.patch,v 1.1 2009/05/10 13:51:45 aballier Exp $
 #
 
@@ -16,9 +16,9 @@
 #10.0.0.0/8
 #172.16.0.0/12
 #192.168.0.0/16
+#169.254.0.0/16
 
-# The world
-# (comment this out to obtain a safe default)
-::/0
-0.0.0.0/0
+# The world (uncommenting these 2 lines is not quite safe)
+#::/0
+#0.0.0.0/0
 
